### PR TITLE
[Administration] & [Get Started] - Content Collections for 24.4.2 by AN

### PR DESCRIPTION
**Pull request by Aishbn**
This PR contains cc updates for the following tickets-
[RLP-133171](https://redlock.atlassian.net/issues/RLP-133171)  -  Jakarta Stack updates.
[RLP-135264](https://redlock.atlassian.net/browse/RLP-135264) - Support for Cortex XSOAR V8 integration.

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- After: https://<branch>--prisma-cloud-docs--hlxsites.hlx.page/
- Worker: https://prisma-cloud-docs-production.adobeaem.workers.dev/?branch=<branch>

### CODE REVIEW

File: docs/en/enterprise-edition/content-collections/administration/configure-external-integrations-on-prisma-cloud/integrate-prisma-cloud-with-cortex-xsoar.adoc [hunk 1]
```diff
@@ -21,40 +21,62 @@ Currently, this integration does not support the use of notification templates a
...
+
+. Select *Add Integration > Cortex XSOAR*. A modal window opens where you can add the details for Cortex integration.
 
-. Enter *Integration Name* and *Description*.
+. Enter the *Integration Name* and an optional *Description*.
 
-. Enter your *Cortex XSOAR Instance FQDN/IP* address.
+. Choose either *Xsoar version 6.0* or *Xsoar version 8.0* as the *Version Available* for integration with Prisma Cloud.
++
+.. For Xsoar version 6.0, add the following details:
```
Comment (jenjoe22) [inline docs/en/enterprise-edition/content-collections/administration/configure-external-integrations-on-prisma-cloud/integrate-prisma-cloud-with-cortex-xsoar.adoc:37]: XSOAR should be all caps here

File: docs/en/enterprise-edition/content-collections/administration/configure-external-integrations-on-prisma-cloud/integrate-prisma-cloud-with-cortex-xsoar.adoc [hunk 2]
```diff
@@ -21,40 +21,62 @@ Currently, this integration does not support the use of notification templates a
...
+The API key you provide must belong to a Cortex XSOAR administrative user with read-write permissions required to enable this push-based integration. Within Cortex XSOAR, navigate to *Settings & Info > Settings > Integrations > API Keys* and get your API key.
+
+... Enter the *API Key ID* which is *ID* number of the API Key. 
 +
-image::administration/get-status.png[]
+image::administration/cortex-xsoar-int-2.png[]
+
+. Click *Next*, review the *Summary* and then *Test Integration*.
+
+. *Save Integration* after the testing is successful.
```
Comment (jenjoe22) [inline docs/en/enterprise-edition/content-collections/administration/configure-external-integrations-on-prisma-cloud/integrate-prisma-cloud-with-cortex-xsoar.adoc:67]: Suggest:

*Save Integration* if the test is successful.

File: docs/en/enterprise-edition/content-collections/administration/configure-external-integrations-on-prisma-cloud/integrate-prisma-cloud-with-cortex-xsoar.adoc (hunk 1 above)
Comment (Aishbn) [reply docs/en/enterprise-edition/content-collections/administration/configure-external-integrations-on-prisma-cloud/integrate-prisma-cloud-with-cortex-xsoar.adoc:37]: XSOAR is in lower case in the UI

File: docs/en/enterprise-edition/content-collections/administration/configure-external-integrations-on-prisma-cloud/integrate-prisma-cloud-with-cortex-xsoar.adoc [hunk 3]
```diff
@@ -21,40 +21,62 @@ Currently, this integration does not support the use of notification templates a
...
+If you are adding a Cortex XSOAR instance that is part of a multi-tenant deployment, enter the tenant URL without the protocol (http or https). For example, enter `<yourcortexxsoarinstance>.com` without \https://www.api- or \https://www.
 
-. Enter the *API Key* associated with the Cortex XSOAR administrative user account.
+... Enter the *API Key* associated with the Cortex XSOAR administrative user account.
 +
-The API key you provide must belong to a Cortex XSOAR administrative user who has read-write permissions, which are required to enable this push-based integration. Within Cortex XSOAR, navigate to *Settings > Integrations > API Keys* and *Get Your Key*.
+The API key you provide must belong to a Cortex XSOAR administrative user with read-write permissions required to enable this push-based integration. Within Cortex XSOAR, navigate to *Settings > Integrations > API Keys* and *Get Your Key*.
 
-. Click *Next* and then *Test*.
+.. For Xsoar version 8.0, add the following details:
```
[the remark this answers is not in the document]
Comment (Aishbn) [reply docs/en/enterprise-edition/content-collections/administration/configure-external-integrations-on-prisma-cloud/integrate-prisma-cloud-with-cortex-xsoar.adoc:49]: XSOAR is in lower case in the UI

**Pull request merged**